hexrd/ui/calibration/auto/instrument_calibrator.py

```diff
@@ -64,22 +155,25 @@ def run_calibration(self,
                     lm = Minimizer()  # noqa: F841
             else:
                 x1, cox_x, infodict, mesg, ierr = leastsq(
-                    obj_func, x0, args=(data_dict, ),
+                    self.residual,
+                    x0, args=(master_data_dict_list, ),
                     full_output=True
                 )
-            resd1 = obj_func(x1, data_dict)
+            resd1 = self.residual(x1, master_data_dict_list)
 
             delta_r = sum(resd0**2)/float(len(resd0)) - \
                 sum(resd1**2)/float(len(resd1))
 
             if delta_r > 0:
-                print('OPTIMIZATION SUCCESSFUL\nfinal ssr: '
-                      f'{sum(resd1**2)/float(len(resd1))}')
-                print(f'delta_r: {delta_r}')
+                print('OPTIMIZATION SUCCESSFUL!!!')
             else:
                 print('no improvement in residual!!!')
-                step_successful = False
+            print('initial ssr: '
+                  f'{sum(resd0**2)/float(len(resd0))}')
+            print('final ssr: '
+                  f'{sum(resd1**2)/float(len(resd1))}')
 
+            step_successful = False
```

Comment:
Looks like this is causing the loop to end after 1 iteration, is that correct?

Reply:
bad indent; fixed.